apps/ui-library/__registry__/index.tsx

```diff
@@ -295,6 +295,18 @@ export const Index: Record<string, any> = {
       chunks: []
     }
     ,
+    "infinite-query-hook": {
```

Comment:
wonder if it should be called "infinite list" (what it is in user-land) vs "infinite query hook" (what it technically is)